javascript/net/grpc/web/grpc_generator.cc

```diff
@@ -290,16 +323,22 @@ class GrpcCodeGenerator : public CodeGenerator {
 
     string file_name;
     string mode;
+    string import_style_str;
+    ImportStyle import_style;
+  
     for (uint i = 0; i < options.size(); ++i) {
```

Comment:
@stanley-cheung This is causing an error for me. Should this not be `size_t`?

Reply:
Ah right. Sorry please change.

Reply:
Done!